Clarify how redacted_because actually works for events (#3411)
matrix-org/matrix-doc: content/client-server-api/_index.md

not required by the protocol. This stripped down event is thereafter
returned anytime a client or remote server requests it. Redacting an
event cannot be undone, allowing server owners to delete the offending
content from the databases. Events that have been redacted should
include a copy of the `m.room.redaction` event under `unsigned` as

> `redacted_because`.

Reviewer: we should probably specify that this is added by the local homeserver, rather than on the event itself?

> `redacted_because` by the local server (not over federation).

Reviewer: I now don't really think this sentence makes any sense :/

Author: would it make more sense with `s/by/from`? or does the order need to be flipped?

> content from the databases. Events that have been redacted should
> include a copy of the `m.room.redaction` event under `unsigned` as
> `redacted_because` by the local server (not over federation).

Reviewer: ```suggestion
content from the databases. Servers should include a copy of the 
`m.room.redaction` event under `unsigned` as `redacted_because` 
when serving the redacted event to clients.
```
